velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -582,5 +583,27 @@ TEST_F(LogNTest, log10) {
   EXPECT_EQ(log10(kInf), kInf);
 }
 
+TEST_F(LogNTest, log) {
+  const auto log = [&](std::optional<double> a, std::optional<double> b) {
+    return evaluateOnce<double>("log(c0, c1)", a, b);
+  };
+  const auto isNan = [&](std::optional<double> res) {
+    return std::isnan(res.value());
+  };
+  EXPECT_EQ(log(10, 100), 2.0);
+  EXPECT_EQ(log(0.0, 1.0), std::nullopt);
+  EXPECT_EQ(log(1.0, 0.0), std::nullopt);
+  EXPECT_EQ(log(-1.0, 1.0), std::nullopt);
+  EXPECT_EQ(log(1.0, -1.0), std::nullopt);
+  EXPECT_EQ(log(1.0, 3.0), kInf);
+  EXPECT_EQ(log(1.0, 3.0), kInf);
+  EXPECT_EQ(log(1.0, 3.0), kInf);
```

Comment:
Please remove duplicate tests for log(1.0, 3.0).

Reply:
Good catch, thanks.